### ISSUE TITLE
Please remove Repo

### ISSUE THREAD

**Issue by radioactiv** [robwink/davinci#3]

**Comment by radioactiv** [comment]
Arizona State University would like to minimize the confusion on the authoritative source for the distribution of grant funded software.  
The license provided under the open source Davinci only allows use of the code, not redistribution.

**Comment by radioactiv** [comment]
2. You may modify your copy or copies of the Program or any portion
of it, thus forming a work based on the Program, and copy and
distribute such modifications or work under the terms of Section 1
above, provided that you also meet all of these conditions:

    a) You must cause the modified files to carry prominent notices
    stating that you changed the files and the date of any change.

    b) You must cause any work that you distribute or publish, that in
    whole or in part contains or is derived from the Program or any
    part thereof, to be licensed as a whole at no charge to all third
    parties under the terms of this License.

    c) If the modified program normally reads commands interactively
    when run, you must cause it, when started running for such
    interactive use in the most ordinary way, to print or display an
    announcement including an appropriate copyright notice and a
    notice that there is no warranty (or else, saying that you provide
    a warranty) and that users may redistribute the program under
    these conditions, and telling the user how to view a copy of this
    License.  (Exception: if the Program itself is interactive but
    does not normally print such an announcement, your work based on
    the Program is not required to print an announcement.)

**Comment by jlaura** [comment]
@radioactiv If the license does not allow redistribution I would contact GitHub directly to let them know that this repository is in violation of this project's license and let them handle it under their terms of service.

**Comment by robwink** [comment]
> Arizona State University would like to minimize the confusion on the authoritative source for the distribution of grant funded software. The license provided under the open source Davinci only allows use of the code, not redistribution.

What license?  Is it not licensed under the GPL 2 which as I understand it gives me the right to redistribute under the same license (or anything even less restrictive)?  Also I specifically remember there being a fork of davinci even 6 years ago (that dated back years before that) that you had no problem with.  It was a significantly changed fork with it's own name and loads of new features but like this, it would have been fine under the GPL, and iirc, it also linked back to the official davinci page.

> Arizona State University would like to minimize the confusion on the authoritative source for the distribution of grant funded software. The license provided under the open source Davinci only allows use of the code, not redistribution.

How is this even a problem?  There is no link to my repo from any official ASU/MSFF/Davinci website.  The only way they are finding my repo is if they are specifically looking for it, or more likely, specifically searching Github for a mirror because svn is terrible and they don't want to deal with it (or git-svn if they've even heard of that).

In any case, even aside from the fact that my very first sentence in the README and README.md says this is a fork not a mirror, and go on to explain the differences, it's obvious that this is not the "official" source for any flavor of davinci.

Why do I care, besides the principle of the thing?

Well, after I left you guys (Saadat I assume) reverted **_all of my work_**.  Not the last month or so where I made extensive changes (like running clang-format on the whole thing because the formatting was driving me nuts), but **_all_** 7 or so months of it.  It made zero sense.  Even if he disagreed with some of the more comprehensive changes and improvements I made later on, the first 2-3 months I was there I did nothing but bug fixes and urgently requested features. An example is the HDF5 work which was discussed ad nauseam so everyone would be happy with the way I maintained backward compatibility with the incorrect way Davinci used to handle HDF files.  Even later in my time, I was almost exclusively working on things that were requested like supporting the new types and adding some much needed test cases and a much nicer testing system using Python.  But no, rather than do the logical thing and actually keep some of the work we paid for, let's just wipe out everything Robert did because ... I didn't like the last couple commits and I'm too lazy to actually look at even the commit messages before those?  Because of pride?  I honestly can't fathom the reasoning that went into that decision.  And as I suspected the level of work on Davinci since I left has gone back to almost 0 and few if any of the fixes and improvements I made ever got reapplied.

Why wouldn't I want to make all the work I did actually available, even if there's only a handful of people who actually use Davinci?  Why wouldn't I want all that time I spent to amount to something?  Why shouldn't every Davinci user have the option, in the case where the Official version is broken, buggy, or lacking features they want, to at least try this fork of Davinci?  I know there were people who were using my version while I was there, several specifically because of certain issues, and some, just to try out the latest features (like the tab completion I added in the later months).  I remember hearing appreciation from those people.  Why shouldn't they be allowed to switch between this and the official version if they want to?  If they're using davinci at all, and especially if they know enough to find and clone and build this repo, they know what they're doing enough to understand the issues involved.

As I said, anyone who finds this was specifically looking for it or something like it, and it is clearly marked as a fork.  The most I would consider is making that even more clear, and maybe changing the name of the repo ... to fork_of_davinci or davinci_fork or something.

Wait, hold the phone.  I was just going to add a PS about there already being a [fork](https://github.com/GEOGUXM/davinci) of this repo so you'd have to get them taken down too if you're committed to this, when I saw my traffic insights, I'm getting traffic from davinci.asu.edu ... what the heck?

I stand corrected, there is still a link from your site to this repo [here](http://davinci.asu.edu/index.php?title=Building_Davinci_for_Windows).  Why are you complaining to me when you've left that up on your wiki all this time?  Remove that and you'd probably solve the issue.  Did you even ask people how they were getting here? smh.
